### PR TITLE
[Merged by Bors] - doc(GroupTheory/Commensurable): clarify some docstrings

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

I was trying to learn about commensurators from mathlib, but all the docstrings seemed to have been written for people who already knew what the definitions were. This wasn't hard to fix.

### CODE REVIEW

File: Mathlib/GroupTheory/Index.lean [hunk 1]
```diff
@@ -42,16 +42,16 @@ open Cardinal
 
 variable {G G' : Type*} [Group G] [Group G'] (H K L : Subgroup G)
 
-/-- The index of a subgroup as a natural number, and returns 0 if the index is infinite. -/
+/-- The index of a subgroup as a natural number. Returns `0` if the index is infinite. -/
 @[to_additive "The index of a subgroup as a natural number,
 and returns 0 if the index is infinite."]
```
Comment (adomani) [inline Mathlib/GroupTheory/Index.lean:47]:
```suggestion
 @[to_additive "The index of a subgroup as a natural number.
Returns 0 if the index is infinite."]
```
I'm not sure that the suggestion will work, as it is outside of the modified lines.

**Review by adomani — COMMENTED** [review]
Thanks: this is much clearer!

bors d+

File: Mathlib/GroupTheory/Index.lean (hunk 1 above)
Comment (kbuzzard) [reply Mathlib/GroupTheory/Index.lean]: I applied it manually.

**Comment by kbuzzard** [comment]
Sorry for the delay, I was travelling last week.

bors r+

**Pull request closed without merging**